oncotarget.csl

```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="en-US">
+  <info>
+    <title>Oncotarget</title>
+    <id>http://www.zotero.org/styles/oncotarget</id>
+    <link href="http://www.zotero.org/styles/oncotarget" rel="self"/>
+    <link href="http://www.zotero.org/styles/vancouver" rel="template"/>
+    <link href="http://www.impactjournals.com/oncotarget" rel="documentation"/>
+    <link href="http://www.nlm.nih.gov/bsd/uniform_requirements.html" rel="documentation"/>
```

Comment:
Can you delete `<link href="http://www.nlm.nih.gov/bsd/uniform_requirements.html" rel="documentation"/>` as well?